engine/src/flutter/impeller/display_list/canvas.cc

```diff
@@ -892,16 +893,16 @@ void Canvas::ClipGeometry(const Geometry& geometry,
   entity.SetClipDepth(clip_depth);
 
   GeometryResult geometry_result = geometry.GetPositionBuffer(
-      renderer_,                                                   //
-      entity,                                                      //
-      *render_passes_.back().inline_pass_context->GetRenderPass()  //
+      renderer_,                                                      //
+      entity,                                                         //
+      *render_passes_.back().GetInlinePassContext()->GetRenderPass()  //
```

Comment:
This sequence is repeated a lot, is it worth a convenience method?

Reply:
Or at least the first 3 parts of this sequence since sometimes the last method is different.

Reply:
It's not a bad idea, but isn't required for this change though so I left it as is.